### PR TITLE
Add use leave page.

### PR DESCRIPTION
**Pull request by maksim-luzin**

### CODE REVIEW

File: frontend/src/components/LeavePage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { FC, useState } from 'react';
+import { Prompt } from 'react-router-dom';
+import ModalWindow from '../ModalWindow';
+
+interface IUseLeavePage {
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/LeavePage/index.tsx]: In the app, we don't use the I prefix for interfaces. Rename to Props.

File: frontend/src/components/LeavePage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        title=""
+        body={message}
+        footer=" "
+        onHide={() => setShowModal(false)}
+      />
+    </>
+  );
+};
+
+export default LeavePage;
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/LeavePage/index.tsx:63]: Please provide example. Maybe we can use this component as Container and path the children here, eg:

```
<LeavePage {...props}>
  <CreateProject />
</LeavePage >
```

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/components/LeavePage/index.tsx (hunk 2 above)
Comment (maksim-luzin) [reply frontend/src/components/LeavePage/index.tsx:63]: This is an example of use.
    <LeavePage
      message="Your message when blocking. Default 'You must completely create a project.'"
      isBlocking={true/false } Default true.
    >
      <CreateProject />
    </LeavePage >

File: frontend/src/components/LeavePage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  /*
+    * This is an example of use.
+    *<LeavePage
+    *  message="Your message when blocking. Default 'You must completely create a project.'"
+    *  isBlocking={true/false } Default true.
+    *>
+    *  <CreateProject />
+    *</LeavePage >
+  */
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/LeavePage/index.tsx]: Please move it to Line 9 and use Doc comments https://typedoc.org/guides/doccomments/

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

**Pull request merged**